cupy/cuda/driver.pyx

```diff
@@ -227,3 +257,76 @@ cpdef launchKernel(
             shared_mem_bytes, <Stream>stream,
             <void**>kernel_params, <void**>extra)
     check_status(status)
+
+
+cpdef FuncAttributes funcGetAttributes(size_t func):
```

Comment:
I'd like to keep `driver.pyx` as a simple thin wrapper (1-to-1 mapping) to CUDA Driver API.

Reply:
Okay, I have now replaced this with a simple wrapper `funcGetAttribute` around `cuFuncGetAttribute`.